lib/jit_preloader/version.rb

```diff
@@ -1,3 +1,3 @@
 module JitPreloader
-  VERSION = "3.1.0"
+  VERSION = "8.0.0"
```

Comment:
Any reason we're jumping versions from 3 -> 8 for this?

Reply:
to keep it in sync with rails, because this gem patches ar. it's simpler to then update it